src/SprykerEco/Zed/PunchoutCatalogs/PunchoutCatalogsDependencyProvider.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs;
+
+use Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnectionQuery;
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerEco\Zed\PunchoutCatalogs\Dependency\Service\PunchoutCatalogsToUtilDateTimeServiceBridge;
+
+/**
+ * @method \SprykerEco\Zed\PunchoutCatalogs\PunchoutCatalogsConfig getConfig()
+ */
+class PunchoutCatalogsDependencyProvider extends AbstractBundleDependencyProvider
+{
+    public const PROPEL_QUERY_PUNCHOUT_CATALOG_CONNECTION = 'QUERY_CONTAINER_CUSTOMER';
```

Comment:
@asmarovydlo Why do you have a different constant name and constant value? They are should be the same.